src/chainstate/burn/db/sortdb.rs

```diff
@@ -2089,15 +2117,55 @@ impl SortitionDB {
             first_block_height,
             first_burn_hash,
             get_epoch_time_secs(),
+            &StacksEpoch::unit_test(first_block_height),
             true,
         )
     }
 
+    /// Validate all Stacks Epochs. Since this is data that always comes from a static variable,
+    /// any invalid StacksEpoch structuring should result in a runtime panic.
+    fn validate_epochs(epochs_ref: &[StacksEpoch]) -> Vec<StacksEpoch> {
+        // sanity check -- epochs must all be contiguous, each epoch must be unique,
+        // and the range of epochs should span the whole u64 space.
```

Comment:
The whole i64 space? (based on `assert_eq!(epoch_end_height, i64::MAX as u64);`)

Reply:
Fixed